CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 ## untagged
 
+There is a new required setting in `chatmail.ini`: `mtail_address`.
+This defines the address on which [`mtail`](https://google.github.io/mtail/)
+exposes its metrics collected from the logs.
+If you want to collect the metrics with Prometheus,
+setup a private network (e.g. WireGuard interface)
+and assign an IP address from this network to the host.
+If unsure, set this setting to `127.0.0.1`.
```

Comment:
why is it a required setting?  It should be fine to leave it empty, and then no mtail is setup or run?